tools/dataproc-edge-node/util/config-files

```diff
@@ -0,0 +1,25 @@
+/etc/hadoop/conf/core-site.xml
+/etc/hadoop/conf/hdfs-site.xml
+/etc/hadoop/conf/mapred-site.xml
+/etc/hadoop/conf/nodes_include
+/etc/hadoop/conf/yarn-site.xml
+/etc/hive/conf/hive-site.xml
+/etc/spark/conf/spark-defaults.conf
+/etc/tez/conf/tez-site.xml
+/etc/zookeeper/conf/zoo.cfg
+/etc/alternatives/hadoop-conf/core-site.xml
+/etc/alternatives/hadoop-conf/hdfs-site.xml
+/etc/alternatives/hadoop-conf/mapred-site.xml
+/etc/alternatives/hadoop-conf/nodes_include
+/etc/alternatives/hadoop-conf/yarn-site.xml
+/etc/alternatives/hive-conf/hive-site.xml
+/etc/alternatives/spark-conf/spark-defaults.conf
+/etc/alternatives/zookeeper-conf/zoo.cfg
+/etc/hadoop/conf.empty/core-site.xml
```

Comment:
Are these actually needed? specifically under /etc/hadoop/conf.empty and /etc/alternatives/hadoop-conf

Reply:
the others are actually symlinks to /etc/hadoop/conf.empty